src/main/java/org/heigit/ohsome/ohsomeapi/utils/RequestUtils.java

```diff
@@ -46,7 +46,9 @@ public static boolean cacheNotAllowed(String url, String[] timeParameter) {
   }
 
   /**
-   * Checks if the given request is requesting a data-extraction.
+   * Checks if the given request is requesting a data extraction that can potentially return a 
```

Comment:
Strange, there was a trailing space at the end of this line (and a few more). I remove them now.